wgpu-core/src/device/mod.rs

```diff
@@ -81,6 +85,8 @@ pub fn all_image_stages() -> hal::pso::PipelineStage {
 }
 
 #[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "trace", derive(Serialize))]
```

Comment:
other places in this module use full paths, like `serde::Deserialize`. Let's make this consistent (one way or another, I think full path is simpler here).